Performance_Testing/Syncfusion_HTMLtoPDF/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,13 @@
+﻿@{
+    ViewData["Title"] = "Home Page";
+}
+
+@{
+    Html.BeginForm("ExportToPDF", "Home", FormMethod.Post);
```

Comment:
Can we change the method name to like ConvertToPdf instead of ExportToPdf?

Reply:
I have changed the method name and ensured the conversion.